articles/how-to-insert-3d-objects-into-a-webpage-using-html-and-css/index.md

```diff
@@ -0,0 +1,320 @@
+    
+</head>
+<body>
+    
+</body>
+</html>
+```
+Start the live server by either right-clicking on the "index.html" file in the explorer and selecting "Open with Live server" or on the status bar clicking on "Go live" to start the server. It will automatically open a preview of the webpage in the default browser which will be blank because we haven't added any content yet. I recommend you use chrome since it supports most 3D objects. You can change the default browser in the settings. If it doesn't automatically open the webpage preview in the browser, check the port number in which the server uses at the Visual Studio Code status bar then add access it in the browser by typing in the URL:
+
+~~~~
+http://127.0.0.1:5501/
+~~~~
+
+Port numbers vary from one to another. In my case, it is port number 5501. If the server doesn't launch completely, stop any services with a similar address running in the background. If it still has errors please look for additional support online concerning the error it displays.
+
+
+In the head section, add a link to import a model-viewer JavaScript code. This enables you to import the component into the webpage.
+
+```html
+    <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
+    <script nomodule src="https://unpkg.com/@google/model-viewer/dist/model-viewer-legacy.js"></script>
+```
+
+Copy your logo picture in the assets folder.
+When done, add a div element of id "container" to hold all our content in the body tags and also another in it for our navigation bar. Let us also add div elements with the following id; "aside" for holding our 3D object, "content" for some explanations on the object, and "icons" for some social media icons. Add a simple logo and some links to the webpage and some of their properties in css as shown below:
+
+```html
+<div id="container">
+    <div id="navbar">
+            <div id="logo">
+                <img src="assets/logo.png" alt="logo">
+            </div>
+            <ul>
+                <li class="active">
+                    <a href="#">Home</a>
+                </li>
+                <li><a href="#">Tech</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+                <li><a href="#">Help</a></li>
+            </ul>
+        </div>
+        <div id="aSide">
+            
+        </div>
+        <div id="content">
+            <h2>3D VR LIVE</h2>
+            <p>The modern headsets, boost your pleasure to the MAX!</p>
+            <button>Shop</button>
+        </div>
+        <div id="icons">
+            <div id="iconsLogo">
+                <i class="fab fa-facebook-f"></i>
+                <i class="fab fa-instagram"></i>
+                <i class="fab fa-youtube"></i>
+                <i class="fab fa-twitter"></i>
+            </div>
+        </div>
+</div>
+```
+
+Now use CSS to style the page and the div elements:
+
+```css
```

Comment:
Instead of putting the whole CSS code here. You can put here the major ones, and remaining you could link them into a github repo containing the code.

Also, don't forget to add comments.

Reply:
Just a quick question, I wanted people to follow along quickly without opening other websites; can't I just leave the css code there so that they can easily copy-paste while learning?

Reply:
@RisoriTofa 

Good question. Yep, you're right.

But, the thing is, the code is really too long. You can break them down into smaller snippets with code explanations added in between. (I understand there isn't much to explain CSS codes, but just explain what the styling does or which HTML element is the styling applied for).

Hope it's clear.

Reply:
I understand
Can I send you the MD so that you can see the changes?
And should I add the link to the repository at the top or bottom section?

Reply:
@RisoriTofa
Not needed, I can see the changes that you've made via your commits. Make sure you push the latest changes here.

You could add the link at the bottom.

Reply:
Ok.
What about the plagiarism screenshots. Do I have to add them, and if so where? (Is it in the article folder?)

Reply:
@RisoriTofa
You can add the screenshot as a new comment. (Not under this suggestion)

Reply:
A comment within the index.md or in github

Reply:
@RisoriTofa

I meant the GitHub, not index.md. Don't push them with your commits.

Comment like [this](https://github.com/section-engineering-education/engineering-education/pull/1766#issuecomment-802887995)

Reply:
I have attached them to the article folder
they are 3 pdfs and one screenshot
They all begin with the name plagiarism, you can check then delete

Reply:
Am done!

Reply:
Great